lib/parser.js

```diff
@@ -66,7 +66,10 @@ Parser.prototype.parse = function(src) {
     comment: true,
     attachComment: true,
     loc: true,
-    ecmaVersion: 6
+    ecmaVersion: 6,
+    ecmaFeatures: {
+      jsx: true
```

Comment:
Enabling JSX by default is a bit aggressive, but it's ok! :smiling_imp: 
We need `.fixclosurerc.json` feature to load such a nested options easier.